projects/batfish/src/main/java/org/batfish/representation/juniper/Route6FilterLineLonger.java

```diff
@@ -18,12 +18,14 @@ public Route6FilterLineLonger(Prefix6 prefix) {
   @Override
   public void applyTo(Route6FilterList rfl) {
     int prefixLength = _prefix6.getPrefixLength();
-    if (prefixLength >= 32) {
-      throw new BatfishException("Route filter prefix length cannot be 'longer' than 32");
+    if (prefixLength >= Prefix6.MAX_PREFIX_LENGTH) {
+      throw new BatfishException(
+          "Route filter prefix length cannot be 'longer' than 128");
```

Comment:
I hate to do this to you, but please change `128"` in the `BatfishException` text to `" + Prefix6.MAX_PREFIX_LENGTH` :)
**DO THIS AFTER THE FORCE PUSH I DESCRIBE BELOW**